db/migration/V021__create_assessments.sql

```diff
@@ -0,0 +1,137 @@
+CREATE TABLE question_types (
+  id BIGINT NOT NULL AUTO_INCREMENT,
+  title TEXT NOT NULL,
+  PRIMARY KEY (id)
+);
+
+CREATE TABLE assessment_submission_states (
+  id BIGINT NOT NULL AUTO_INCREMENT,
+  title TEXT NOT NULL,
+  PRIMARY KEY (id)
+);
+
+CREATE TABLE program_participant_roles (
+  id BIGINT NOT NULL AUTO_INCREMENT,
+  title TEXT NOT NULL,
+  PRIMARY KEY (id)
+);
+
+CREATE TABLE curriculum_assessments (
+  id BIGINT NOT NULL AUTO_INCREMENT,
+  title TEXT NOT NULL,
+  description TEXT,
+  max_score INT NOT NULL,
+  max_num_submissions INT NOT NULL,
+  time_limit INT,
+  curriculum_id BIGINT NOT NULL,
+  activity_id BIGINT NOT NULL,
+  principal_id BIGINT NOT NULL,
+  created_at TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP,
+  updated_at TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP ON UPDATE CURRENT_TIMESTAMP,
+  INDEX curriculum_assessments_curriculum_id (curriculum_id),
+  FOREIGN KEY (curriculum_id) REFERENCES curriculums(id),
+  INDEX curriculum_assessments_activity_id (activity_id),
+  FOREIGN KEY (activity_id) REFERENCES activities(id),
```

Comment:
This should have an `ON DELETE SET NULL`.

Reply:
added ON DELETE SET NULL